<!DOCTYPE html>
<html lang="en" dir="ltr" itemscope itemtype="http://schema.org/Article" data-r-output-format="html">
  <head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
    <meta charset="utf-8">
    <meta name="viewport" content="height=device-height, width=device-width, initial-scale=1.0, minimum-scale=1.0">
    <meta name="generator" content="Hugo 0.150.0">
    <meta name="generator" content="Relearn 8.0.1+b23cf6629eada0c2802f34ae4012e04343497862">
    <meta name="description" content="Enabling Rapid Genomic and Multiomic Data Analysis with Illumina DRAGEN‚Ñ¢ v4.4 on Amazon EC2 F2 Instances Eric Allen, Mark Azadpour, Deepthi Shankar, Olivia Choudhury, and Shyamal Mehtalia | 15/07/2025 | High Performance Computing, Life Sciences, Partner solutions
This post was contributed by Eric Allen (AWS), Olivia Choudhury (AWS), Mark Azadpour (AWS), Deepthi Shankar (Illumina), and Shyamal Mehtalia (Illumina)
The analysis of ever-increasing amounts of genomic and multiomic data demands efficient, scalable, and cost-effective computational solutions. Amazon Web Services (AWS) continues to support these workloads through FPGA accelerated compute offerings such as Amazon EC2 F2 instances.">
    <meta name="author" content="">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:title" content=" Blog 8">
    <meta name="twitter:description" content="Enabling Rapid Genomic and Multiomic Data Analysis with Illumina DRAGEN‚Ñ¢ v4.4 on Amazon EC2 F2 Instances Eric Allen, Mark Azadpour, Deepthi Shankar, Olivia Choudhury, and Shyamal Mehtalia | 15/07/2025 | High Performance Computing, Life Sciences, Partner solutions
This post was contributed by Eric Allen (AWS), Olivia Choudhury (AWS), Mark Azadpour (AWS), Deepthi Shankar (Illumina), and Shyamal Mehtalia (Illumina)
The analysis of ever-increasing amounts of genomic and multiomic data demands efficient, scalable, and cost-effective computational solutions. Amazon Web Services (AWS) continues to support these workloads through FPGA accelerated compute offerings such as Amazon EC2 F2 instances.">
    <meta property="og:url" content="http://localhost:1313/3-blogstranslated/3.8-blog8/index.html">
    <meta property="og:title" content=" Blog 8">
    <meta property="og:description" content="Enabling Rapid Genomic and Multiomic Data Analysis with Illumina DRAGEN‚Ñ¢ v4.4 on Amazon EC2 F2 Instances Eric Allen, Mark Azadpour, Deepthi Shankar, Olivia Choudhury, and Shyamal Mehtalia | 15/07/2025 | High Performance Computing, Life Sciences, Partner solutions
This post was contributed by Eric Allen (AWS), Olivia Choudhury (AWS), Mark Azadpour (AWS), Deepthi Shankar (Illumina), and Shyamal Mehtalia (Illumina)
The analysis of ever-increasing amounts of genomic and multiomic data demands efficient, scalable, and cost-effective computational solutions. Amazon Web Services (AWS) continues to support these workloads through FPGA accelerated compute offerings such as Amazon EC2 F2 instances.">
    <meta property="og:locale" content="en">
    <meta property="og:type" content="website">
    <meta itemprop="name" content=" Blog 8">
    <meta itemprop="description" content="Enabling Rapid Genomic and Multiomic Data Analysis with Illumina DRAGEN‚Ñ¢ v4.4 on Amazon EC2 F2 Instances Eric Allen, Mark Azadpour, Deepthi Shankar, Olivia Choudhury, and Shyamal Mehtalia | 15/07/2025 | High Performance Computing, Life Sciences, Partner solutions
This post was contributed by Eric Allen (AWS), Olivia Choudhury (AWS), Mark Azadpour (AWS), Deepthi Shankar (Illumina), and Shyamal Mehtalia (Illumina)
The analysis of ever-increasing amounts of genomic and multiomic data demands efficient, scalable, and cost-effective computational solutions. Amazon Web Services (AWS) continues to support these workloads through FPGA accelerated compute offerings such as Amazon EC2 F2 instances.">
    <meta itemprop="wordCount" content="1720">
    <title> Blog 8</title>
    <link href="http://localhost:1313/vi/3-blogstranslated/3.8-blog8/index.html" rel="alternate" hreflang="x-default">
    <link href="http://localhost:1313/vi/3-blogstranslated/3.8-blog8/index.html" rel="alternate" hreflang="vi">
    <link href="http://localhost:1313/3-blogstranslated/3.8-blog8/index.html" rel="alternate" hreflang="en">
    <link href="/3-blogstranslated/3.8-blog8/index.xml" rel="alternate" type="application/rss+xml" title=" Blog 8">
    <link href="/images/logo.png?1762742815" rel="icon" type="image/png">
    <link href="/css/auto-complete/auto-complete.min.css?1762742815" rel="stylesheet">
    <script src="/js/auto-complete/auto-complete.min.js?1762742815" defer></script>
    <script src="/js/search-lunr.js?1762742815" defer></script>
    <script src="/js/search.js?1762742815" defer></script>
    <script>
      window.relearn = window.relearn || {};
      window.relearn.index_js_url="/searchindex.en.js?1762742815";
    </script>
    <script src="/js/lunr/lunr.min.js?1762742815" defer></script>
    <script src="/js/lunr/lunr.stemmer.support.min.js?1762742815" defer></script>
    <script src="/js/lunr/lunr.multi.min.js?1762742815" defer></script>
    <script src="/js/lunr/lunr.en.min.js?1762742815" defer></script>
    <script>
      window.relearn = window.relearn || {};
      window.relearn.contentLangs=['en'];
    </script>
    <link href="/fonts/fontawesome/css/fontawesome-all.min.css?1762742815" rel="stylesheet" media="print" onload="this.media='all';this.onload=null;"><noscript><link href="/fonts/fontawesome/css/fontawesome-all.min.css?1762742815" rel="stylesheet"></noscript>
    <link href="/css/perfect-scrollbar/perfect-scrollbar.min.css?1762742815" rel="stylesheet">
    <link href="/css/theme.css?1762742815" rel="stylesheet">
    <link href="/css/format-html.css?1762742815" rel="stylesheet" id="R-format-style">
    <script>
      window.relearn = window.relearn || {};
      // configuration
      window.relearn.min = ``;
      window.relearn.path='\/3-blogstranslated\/3.8-blog8\/index.html';
      window.relearn.relBasePath='..\/..';
      window.relearn.relBaseUri='..\/..';
      window.relearn.absBaseUri='http:\/\/localhost:1313';
      window.relearn.disableAnchorCopy=false;
      window.relearn.disableAnchorScrolling=false;
      window.relearn.disableInlineCopyToClipboard=false;
      window.relearn.enableBlockCodeWrap=true;
      // legal
      window.relearn.getItem = (s,n) => {return s.getItem(n)};
      window.relearn.setItem = (s,n,v) => {return s.setItem(n,v)};
      window.relearn.removeItem = (s,n) => {return s.removeItem(n)};
      // translations
      window.T_Copy_to_clipboard = `Copy to clipboard`;
      window.T_Copied_to_clipboard = `Copied to clipboard!`;
      window.T_Copy_link_to_clipboard = `Copy link to clipboard`;
      window.T_Link_copied_to_clipboard = `Copied link to clipboard!`;
      window.T_Reset_view = `Reset view`;
      window.T_View_reset = `View reset!`;
      window.T_No_results_found = `No results found for "{0}"`;
      window.T_N_results_found = `{1} results found for "{0}"`;
      // variant stuff
      window.relearn.themevariants = [ 'auto' ];
      window.relearn.customvariantname = "my-custom-variant";
      window.relearn.changeVariant = function(variant) {
        var oldVariant = document.documentElement.dataset.rThemeVariant;
        window.relearn.setItem(window.localStorage, window.relearn.absBaseUri + "/variant", variant);
        document.documentElement.dataset.rThemeVariant = variant;
        if (oldVariant != variant) {
          document.dispatchEvent( new CustomEvent('themeVariantLoaded', { detail: { variant, oldVariant } }) );
          window.relearn.markVariant();
        }
      }
      window.relearn.markVariant = function() {
        var variant = window.relearn.getItem(window.localStorage, window.relearn.absBaseUri + "/variant");
        document.querySelectorAll(".R-variantswitcher select").forEach((select) => {select.value = variant;});
      }
      window.relearn.initVariant = function() {
        var variant = window.relearn.getItem(window.localStorage, window.relearn.absBaseUri + "/variant") ?? "";
        if( variant == window.relearn.customvariantname ){
        }else if( !variant || !window.relearn.themevariants.includes(variant) ){
          variant = window.relearn.themevariants[0];
          window.relearn.setItem(window.localStorage, window.relearn.absBaseUri + "/variant", variant);
        }
        document.documentElement.dataset.rThemeVariant = variant;
      }
      window.relearn.initVariant();
      window.relearn.markVariant();
    </script>
    <link href="/css/custom.css?1762742815" rel="stylesheet">
  </head>
  <body class="mobile-support html" data-url="/3-blogstranslated/3.8-blog8/index.html">
    <div id="R-body" class="default-animation">
      <div id="R-body-overlay"></div>
      <nav id="R-topbar">
        <div class="topbar-wrapper">
          <div class="topbar-sidebar-divider"></div>
          <div class="topbar-area topbar-area-start" data-area="start">
            <div class="topbar-button topbar-button-sidebar" data-content-empty="disable" data-width-s="show" data-width-m="hide" data-width-l="hide"><button class="topbar-control" onclick="toggleNav()" type="button" title="Menu (CTRL&#43;ALT&#43;n)"><i class="fa-fw fas fa-bars"></i></button>
            </div>
            <div class="topbar-button topbar-button-toc" data-content-empty="hide" data-width-s="show" data-width-m="show" data-width-l="show"><button class="topbar-control" onclick="toggleTopbarFlyout(this)" type="button" title="Table of Contents (CTRL&#43;ALT&#43;t)"><i class="fa-fw fas fa-list-alt"></i></button>
              <div class="topbar-content">
                <div class="topbar-content-wrapper">
<nav class="TableOfContents">
  <ul>
    <li><a href="#amazon-ec2-f2-instances-overview">Amazon EC2 F2 instances overview</a></li>
    <li><a href="#performance-benchmarking-approach">Performance benchmarking approach</a></li>
    <li><a href="#speed--cost-performance-comparison-wgs-analysis">Speed &amp; cost performance comparison: WGS analysis</a></li>
    <li><a href="#speed--cost-performance-comparison-tumor-normal-analysis">Speed &amp; cost performance comparison: Tumor Normal analysis</a></li>
    <li><a href="#other-advantages">Other advantages</a></li>
    <li><a href="#f2-instance-availability-and-cloud-implementation-options">F2 instance availability and cloud implementation options</a></li>
    <li><a href="#conclusion">Conclusion</a></li>
    <li><a href="#references">References</a></li>
    <li><a href="#footnotes">Footnotes</a></li>
  </ul>
</nav>
                </div>
              </div>
            </div>
          </div>
          <ol class="topbar-breadcrumbs breadcrumbs highlightable" itemscope itemtype="http://schema.org/BreadcrumbList"><li itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement" class=""><a itemprop="item" href="/index.html"><span itemprop="name">Intership Report</span></a><meta itemprop="position" content="1">&nbsp;>&nbsp;</li><li itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement" class=""><a itemprop="item" href="/3-blogstranslated/index.html"><span itemprop="name">BlogsTranslated</span></a><meta itemprop="position" content="2">&nbsp;>&nbsp;</li><li itemscope itemtype="https://schema.org/ListItem" itemprop="itemListElement" class=""><span itemprop="name"> Blog 8</span><meta itemprop="position" content="3"></li>
          </ol>
          <div class="topbar-area topbar-area-end" data-area="end">
            <div class="topbar-button topbar-button-prev" data-content-empty="disable" data-width-s="show" data-width-m="show" data-width-l="show"><a class="topbar-control" href="/3-blogstranslated/3.7-blog7/index.html" title="Blog 7 (ü°ê)"><i class="fa-fw fas fa-chevron-left"></i></a>
            </div>
            <div class="topbar-button topbar-button-next" data-content-empty="disable" data-width-s="show" data-width-m="show" data-width-l="show"><a class="topbar-control" href="/4-eventparticipated/index.html" title="EventParticipated (ü°í)"><i class="fa-fw fas fa-chevron-right"></i></a>
            </div>
            <div class="topbar-button topbar-button-more" data-content-empty="hide" data-width-s="show" data-width-m="show" data-width-l="show"><button class="topbar-control" onclick="toggleTopbarFlyout(this)" type="button" title="More"><i class="fa-fw fas fa-ellipsis-v"></i></button>
              <div class="topbar-content">
                <div class="topbar-content-wrapper">
                  <div class="topbar-area topbar-area-more" data-area="more">
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </nav>
      <div id="R-main-overlay"></div>
      <main id="R-body-inner" class="highlightable 3-blogstranslated" tabindex="-1">
        <div class="flex-block-wrapper">
<article class="default">
  <header class="headline">
  </header>

<h1 id="blog-8"> Blog 8</h1>

<h1 id="enabling-rapid-genomic-and-multiomic-data-analysis-with-illumina-dragen-v44-on-amazon-ec2-f2-instances"><strong>Enabling Rapid Genomic and Multiomic Data Analysis with Illumina DRAGEN‚Ñ¢ v4.4 on Amazon EC2 F2 Instances</strong></h1>
<p><em>Eric Allen, Mark Azadpour, Deepthi Shankar, Olivia Choudhury, and Shyamal Mehtalia</em> | 15/07/2025 | <a href="https://aws.amazon.com/blogs/hpc/category/high-performance-computing/" rel="external" target="_blank">High Performance Computing</a>, <a href="https://aws.amazon.com/blogs/hpc/category/industries/life-sciences/" rel="external" target="_blank">Life Sciences</a>, <a href="https://aws.amazon.com/blogs/hpc/category/post-types/partner-solutions/" rel="external" target="_blank">Partner solutions</a></p>
<p>This post was contributed by Eric Allen (AWS), Olivia Choudhury (AWS), Mark Azadpour (AWS), Deepthi Shankar (Illumina), and Shyamal Mehtalia (Illumina)</p>
<p><a href="#R-image-4830e1f9d0f9ab33d894060bbca7efbf" class="lightbox-link"><img alt="image1" class="lazy lightbox figure-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-p1-1024x286.png" style=" height: auto; width: auto;"></a>
<a href="javascript:history.back();" class="lightbox-back" id="R-image-4830e1f9d0f9ab33d894060bbca7efbf"><img alt="image1" class="lazy lightbox lightbox-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-p1-1024x286.png"></a></p>
<p>The analysis of ever-increasing amounts of genomic and multiomic data demands efficient, scalable, and cost-effective computational solutions. Amazon Web Services (AWS) continues to support these workloads through FPGA accelerated compute offerings such as <a href="https://aws.amazon.com/ec2/instance-types/f2/" rel="external" target="_blank">Amazon EC2 F2 instances</a>.</p>
<p>Illumina‚Äôs <a href="https://www.illumina.com/products/by-type/informatics-products/dragen-secondary-analysis.html" rel="external" target="_blank">DRAGEN (Dynamic Read Analysis for GENomics)</a> secondary analysis solution has established itself as a leading secondary analysis solution for next generation sequencing data offering highly optimized algorithms for genomic analysis and hardware-accelerated implementations of comprehensive genomic and multiomic pipelines including germline DNA, somatic DNA, as well as bulk and single cell RNA analysis, proteomics, spatial, and more.</p>
<p>DRAGEN runs natively on EC2 F2 instances and provides customers with an accelerated approach for analyzing their biological data sets. Migration to F2s is streamlined as the same DRAGEN AMI is used for both F1 and F2, and analysis results will be equivalent1. In this post, we will discuss the performance characteristics of DRAGEN across different AWS compute environments as well as how to run <a href="https://investor.illumina.com/news/press-release-details/2025/Illumina-DRAGEN-v4-4-powers-clinical-oncology-research-and-multiomic-applications/default.aspx" rel="external" target="_blank">the recently launched DRAGEN v4.4</a> on Amazon EC2 F2 instances.</p>
<h2 id="amazon-ec2-f2-instances-overview">Amazon EC2 F2 instances overview</h2>
<p>F2 instances are the second generation of FPGA powered EC2 instances designed to accelerate analysis of genomic and multimedia data in the cloud. These instances offer significant improvements over their predecessors, the F1 instances, providing up to 60% better price performance. Here are the key features and specifications of F2 instances:</p>
<ul>
<li>
<p>FPGA Configuration: F2 instances are equipped with up to eight AMD Virtex UltraScale+ HBM VU47P FPGAs, each featuring 16GB of high-bandwidth memory (HBM).</p>
</li>
<li>
<p>Processor: Powered by 3rd generation AMD EPYC (Milan) processors, F2 instances offer up to 192 vCPUs, which is three times the number of processor cores compared to F1 instances.</p>
</li>
<li>
<p>Memory: These instances provide up to 2 TiB of system memory, doubling the memory capacity of F1 instances.</p>
</li>
<li>
<p>Storage: F2 instances come with up to 7.6 TiB of NVMe SSD storage, which is twice the storage capacity of F1 instances.</p>
</li>
<li>
<p>Networking: They offer up to 100 Gbps of networking bandwidth, quadrupling the network bandwidth available in F1 instances.</p>
</li>
</ul>
<table>
  <tr>
    <th>Instance Name</th>
    <th>FPGAs</th>
    <th>vCPUs</th>
    <th>Instance Memory</th>
    <th>NVMe Storage</th>
    <th>Network Bandwidth</th>
  </tr>
  <tr>
    <td>f1.2xlarge</td>
    <td>1</td>
    <td>8</td>
    <td>122</td>
    <td>470</td>
    <td>Up to 10 Gbps</td>
  </tr>
  <tr style="background-color:#A5CEF0;color: black;"> <!-- t√¥ m√†u xanh l√° -->
    <td>f1.4xlarge</td>
    <td>2</td>
    <td>16</td>
    <td>244</td>
    <td>940</td>
    <td>10 Gbps</td>
  </tr>
  <tr style="background-color:#A9D979;color: black;"> <!-- t√¥ m√†u xanh l√° -->
    <td>f2.6xlarge</td>
    <td>1</td>
    <td>24</td>
    <td>256</td>
    <td>950</td>
    <td>10 Gbps</td>
  </tr>
   <tr>
    <td>F2.12xlarge</td>
    <td>2</td>
    <td>48</td>
    <td>512</td>
    <td>1900</td>
    <td>25 Gbps</td>
  </tr>
   <tr>
    <td>f1.16xlarge</td>
    <td>8</td>
    <td>64</td>
    <td>976</td>
    <td>44x940</td>
    <td>25 Gbps</td>
  </tr>
   <tr>
    <td>F2.48xlarge</td>
    <td>8</td>
    <td>192</td>
    <td>2048</td>
    <td>7600</td>
    <td>100 Gbps</td>
  </tr>
</table>
<p><em>Table 1: Table showing comparative compute, memory, storage, and networking specs for F1 vs F2 instances.</em></p>
<h2 id="performance-benchmarking-approach">Performance benchmarking approach</h2>
<p>Illumina recommends using f1.4xlarge when using F1 instances and f2.6xlarge when using F2 instances. To assess performance across these instances, DRAGEN was configured and run on AWS following the <a href="https://help.dragen.illumina.com/product-guides/dragen-v4.4" rel="external" target="_blank">Illumina DRAGEN user guide</a> and the genome reference file links can be found on the <a href="https://support.illumina.com/sequencing/sequencing_software/dragen-bio-it-platform/product_files.html" rel="external" target="_blank">Illumina DRAGEN Product Files web page</a>.</p>
<ol>
<li>
<p>Whole Genome Sequencing (WGS) analysis at approximately 35x depth using a publicly available sample. The sample HG002 from the <a href="https://www.nist.gov/programs-projects/genome-bottle" rel="external" target="_blank">NIST Genome in a Bottle project</a> was used. This sample was analyzed using DRAGEN v4.4 Germline pipeline in two different manners.  The first ‚Äúbasic‚Äù analysis included only basic alignment and small variant calling, to facilitate comparisons with common bioinformatics pipelines for germline samples, such as BWA/GATK.  The second ‚Äúfull‚Äù analysis was with all variant callers (including for copy number and structural variants) and additional options such as pharmacogenetic star allele calling and HLA (Human Leukocyte Antigen) calling turned on, to produce a fully analyzed whole genome.  In both cases, DRAGEN‚Äôs hg38 multigenome graph reference was used for WGS analysis. The sample data fastq files can be accessed on Amazon S3 using these links: <a href="https://s3/ilmn-dragen-giab-samples/WGS/precisionFDA_v2_HG002/HG002.novaseq.pcr-free.35x.R1.fastq.gz" rel="external" target="_blank">fastq R1</a>, <a href="https://s3/ilmn-dragen-giab-samples/WGS/precisionFDA_v2_HG002/HG002.novaseq.pcr-free.35x.R2.fastq.gz" rel="external" target="_blank">fastq R2</a>.</p>
</li>
<li>
<p>Tumor Normal analysis using a pair of samples examined in a prior <a href="https://www.biorxiv.org/content/10.1101/2023.03.23.534011v2" rel="external" target="_blank">DRAGEN cancer analysis publication</a>. These two samples had approximate depths of 110x (tumor) and 40x (normal).  These samples were analyzed using the DRAGEN v4.4 Somatic pipeline, including alignment and small variant calling, plus CNV and SV analysis. DRAGEN‚Äôs hg38 linear genome reference was used for the Tumor Normal analysis. The sample data fastq files can be accessed on Amazon S3 using these links: <a href="https://s3/dragen-wgs-tn-somatic-benchmarking-datasets/fastq/HCC1395_75pc/tumor/T_SRR7890895_75pc_1.fastq.gz" rel="external" target="_blank">Tumor fastq R1</a>, <a href="https://s3/dragen-wgs-tn-somatic-benchmarking-datasets/fastq/HCC1395_75pc/tumor/T_SRR7890895_75pc_2.fastq.gz" rel="external" target="_blank">Tumor fastq R2</a>, <a href="https://s3/dragen-wgs-tn-somatic-benchmarking-datasets/fastq/HCC1395_75pc/normal/N_SRR7890889_1.fastq.gz" rel="external" target="_blank">Normal fastq R1</a>, <a href="https://s3/dragen-wgs-tn-somatic-benchmarking-datasets/fastq/HCC1395_75pc/normal/N_SRR7890889_2.fastq.gz" rel="external" target="_blank">Normal fastq R2</a>.</p>
</li>
</ol>
<h2 id="speed--cost-performance-comparison-wgs-analysis">Speed &amp; cost performance comparison: WGS analysis</h2>
<p>WGS analysis using DRAGEN v4.4 demonstrated significant price performance advantages on Amazon EC2 F2 instances vs F1, while also generating equivalent analysis results between the two instance generations1:</p>
<ul>
<li>
<p>Basic WGS analysis, including alignment and small variant calling. DRAGEN v4.4 analysis on f2.6xlarge had <strong>1.5x the speed</strong> and <strong>40% of the EC2 compute cost</strong> vs f1.4xlarge.</p>
</li>
<li>
<p>Complete WGS analysis, including alignment, small variant calling and calling of CNVs, SVs, and repeat expansions, and variant annotation. DRAGEN analysis on f2.6xlarge had <strong>2x the speed</strong> and <strong>30% of the EC2 compute cost</strong> vs f1.4xlarge.</p>
</li>
</ul>
<p><a href="#R-image-1739a11aea6f49eddd034e697c73556e" class="lightbox-link"><img alt="image2" class="lazy lightbox figure-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f1.png" style=" height: auto; width: auto;"></a>
<a href="javascript:history.back();" class="lightbox-back" id="R-image-1739a11aea6f49eddd034e697c73556e"><img alt="image2" class="lazy lightbox lightbox-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f1.png"></a></p>
<p><em>Figure 1: f2.6xlarge is 1.5x faster for WGS Basic Analysis and 2.1x faster for WGS Full Analysis than f1.4xlarge.</em></p>
<p><a href="#R-image-d5fcab15a9c01a78cd9cd02d7bb8f5ae" class="lightbox-link"><img alt="image3" class="lazy lightbox figure-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f2.png" style=" height: auto; width: auto;"></a>
<a href="javascript:history.back();" class="lightbox-back" id="R-image-d5fcab15a9c01a78cd9cd02d7bb8f5ae"><img alt="image3" class="lazy lightbox lightbox-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f2.png"></a></p>
<p><em>Figure 2: EC2 compute cost on f2.6xlarge is 40% of the cost on f1.4xlarge for WGS Basic Analysis and 30% of the cost on f1.4xlarge for WGS Full Analysis.</em></p>
<h2 id="speed--cost-performance-comparison-tumor-normal-analysis">Speed &amp; cost performance comparison: Tumor Normal analysis</h2>
<p>As with the WGS results, for Tumor Normal analysis DRAGEN v4.4 also demonstrated significant price performance advantages on Amazon EC2 F2 instances vs F1, while also generating equivalent analysis results between the two instance generations1:</p>
<ul>
<li>Tumor Normal analysis, including alignment, small variant calling, and calling of CNVs and SVs. DRAGEN analysis on f2.6xlarge had <strong>1.7x the speed</strong> and <strong>35% of the EC2 compute cost</strong> vs f1.4xlarge.</li>
</ul>
<p><a href="#R-image-9411a8c2dc7fd6447d99a6755bd6afb1" class="lightbox-link"><img alt="image4" class="lazy lightbox figure-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f3.png" style=" height: auto; width: auto;"></a>
<a href="javascript:history.back();" class="lightbox-back" id="R-image-9411a8c2dc7fd6447d99a6755bd6afb1"><img alt="image4" class="lazy lightbox lightbox-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f3.png"></a></p>
<p><em>Figure 3: f2.6xlarge is 1.7x faster than f1.4xlarge for Tumor Normal Analysis.</em></p>
<p><a href="#R-image-44a48a5a665f2eba366e2a4f222452a0" class="lightbox-link"><img alt="image5" class="lazy lightbox figure-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f4.png" style=" height: auto; width: auto;"></a>
<a href="javascript:history.back();" class="lightbox-back" id="R-image-44a48a5a665f2eba366e2a4f222452a0"><img alt="image5" class="lazy lightbox lightbox-image" loading="lazy" src="../../../images/Blog8/HPCBlog-366-f4.png"></a></p>
<p><em>Figure 4: EC2 compute cost on f2.6xlarge is 35% of the cost on f1.4xlarge for WGS Tumor Normal Analysis.</em></p>
<h2 id="other-advantages">Other advantages</h2>
<p>Traditional genomic analysis pipelines using BWA-MEM and GATK running on CPUs have been the industry standard in the past, but DRAGEN has gained traction by providing speed and accuracy advantages. A number of peer reviewed publications have compared DRAGEN speed and accuracy vs BWA/GATK based pipelines running on CPU.  For example, <a href="https://pubmed.ncbi.nlm.nih.gov/36513709/" rel="external" target="_blank">Ziegler et al. (2022)</a> found that DRAGEN analysis on FPGA hardware was over 8x faster and more accurate than BWA/GATK based pipelines running on CPU, while <a href="https://www.nature.com/articles/s41587-024-02382-1" rel="external" target="_blank">Sedlazek et al. (2024)</a> also found DRAGEN to have higher speed and accuracy vs BWA/GATK based pipelines.</p>
<p>Utilizing DRAGEN on F instances, which are powered by FPGAs, also offers power consumption advantages over traditional CPU and GPU solutions. FPGAs are inherently more energy-efficient, consuming less power for the same level of computational performance for these workloads. This is particularly important in genomic data analysis tasks, where the volume of data and amount of processing time required can be immense.</p>
<p>For instance, FPGAs achieve higher performance per watt compared to CPUs and GPUs for DRAGEN WGS workloads. FPGA-based accelerators can deliver superior throughput with lower power consumption. This is due to the dynamic customizability of FPGAs, which allows for optimized configurations that further enhance energy efficiency. In contrast, CPUs and GPUs, while powerful, often require more energy to perform the same tasks, leading to higher operational costs and a larger environmental footprint.</p>
<p>The lower power consumption of FPGAs translates to reduced cooling requirements, which can be a significant cost factor in large-scale computing environments. Additionally, the energy efficiency of FPGAs makes them an attractive option for high-performance computing applications, where scalability and cost-effectiveness are critical.</p>
<p>In summary, leveraging DRAGEN on Amazon EC2 ‚ÄòF‚Äô family instances provides a more energy-efficient solution for genomic data analysis compared to traditional CPU or GPU-based approaches, offering both cost savings and environmental benefits.</p>
<h2 id="f2-instance-availability-and-cloud-implementation-options">F2 instance availability and cloud implementation options</h2>
<p>Amazon EC2 F2 instances are available in multiple AWS Regions, including US East (N. Virginia), US West (Oregon), Europe (London), and Asia Pacific (Sydney), with plans for expanded Regional availability.   They come in different sizes, such as f2.6xlarge, f2.12xlarge and f2.48xlarge, catering to various workload requirements.</p>
<p>When configuring storage and compute for running DRAGEN workloads on F instances on AWS, it‚Äôs important to select the right options to balance performance and cost. Consider storage options such as <a href="https://aws.amazon.com/ebs/general-purpose/" rel="external" target="_blank">Amazon EBS gp3 volumes</a> configured for RAID, <a href="https://aws.amazon.com/fsx/lustre/" rel="external" target="_blank">Amazon FSx for Lustre</a> for higher throughput, and <a href="https://aws.amazon.com/efs/" rel="external" target="_blank">Amazon Elastic File System (EFS)</a> for persistent storage. Additionally, streaming BAM files and reference data from <a href="https://aws.amazon.com/s3/" rel="external" target="_blank">Amazon Simple Storage Service (Amazon S3)</a> or using <a href="https://aws.amazon.com/s3/features/mountpoint/" rel="external" target="_blank">Mountpoint for Amazon S3</a> to mount S3 buckets on the local file system can provide economical and simplified file access. By carefully choosing and configuring these storage solutions, you can ensure optimal performance and cost-effectiveness for your HPC workloads. Additionally, consider usingg <a href="https://www.illumina.com/products/by-type/informatics-products/connected-analytics.html" rel="external" target="_blank">Illumina Connected Analytics (ICA)</a> or <a href="https://aws.amazon.com/batch/" rel="external" target="_blank">AWS Batch</a> for workflow management.  Illumina provides guidance for running <a href="https://help.dragen.illumina.com/reference/dragen-multi-cloud/dragen-on-aws" rel="external" target="_blank">DRAGEN tr√™n AWS</a> as part of their <a href="https://help.dragen.illumina.com/" rel="external" target="_blank">online DRAGEN user guide</a>.</p>
<h2 id="conclusion">Conclusion</h2>
<p>In summary, Amazon EC2 F2 instances represent a significant advancement in FPGA-powered cloud computing, offering enhanced performance, memory, storage, and networking capabilities over the previous generations. The combination of DRAGEN‚Äôs comprehensive pipelines with Amazon EC2 F2‚Äôs improved computational power delivers faster, more efficient processing of complex datasets ‚Äì from whole genome sequencing to single-cell RNA analysis.</p>
<p>Start your move to F2s today. Please reach out to your AWS account team or Illumina for help with your migration to AWS EC2 F2 instances.</p>
<p>For more info about DRAGEN on AWS, search for <a href="https://aws.amazon.com/marketplace/search/results?searchTerms=DRAGEN+Complete+Suite" rel="external" target="_blank">DRAGEN Complete Suite tr√™n AWS Marketplace</a>, view the <a href="https://developer.illumina.com/news-updates/dragen-v4-4" rel="external" target="_blank">blog DRAGEN v4.4</a>, or visit <a href="https://www.illumina.com/products/by-type/informatics-products.html" rel="external" target="_blank">trang ch·ªß Illumina Informatics Solutions</a>.</p>
<h2 id="references">References</h2>
<p>Scheffler, K. et al. ‚ÄúSomatic small-variant calling methods in Illumina DRAGEN‚Ñ¢ Secondary Analysis.‚Äù BioRxiv (2023). <a href="https://www.biorxiv.org/content/10.1101/2023.03.23.534011v2" rel="external" target="_blank">https://www.biorxiv.org/content/10.1101/2023.03.23.534011v2</a></p>
<p>Sedlazeck, F. J. et al. ‚ÄúComprehensive genome analysis and variant detection at scale using DRAGEN.‚Äù Nature Biotechnology (2024). <a href="https://www.nature.com/articles/s41587-024-02382-1" rel="external" target="_blank">https://www.nature.com/articles/s41587-024-02382-1</a></p>
<p>Ziegler, A. et al. ‚ÄúComparison of calling pipelines for whole genome sequencing: an empirical study demonstrating the importance of mapping and alignment.‚Äù Scientific Reports (2022).
<a href="https://pubmed.ncbi.nlm.nih.gov/36513709/" rel="external" target="_blank">https://pubmed.ncbi.nlm.nih.gov/36513709/</a></p>
<h2 id="footnotes">Footnotes</h2>
<p>Equivalence between F1 and F2 hard-filtered.vcf and other results files based on analysis results using DRAGEN 4.4.4 following Illumina DRAGEN usage instructions. The vim diff command shows the only difference between the two VCF files is an entry showing the analysis execution time.  Customers can do similar analyses themselves to verify or reach out to Illumina for more information.</p>

  <footer class="footline">
  </footer>
</article>
        </div>
      </main>
    </div>
    <aside id="R-sidebar" class="default-animation">
      <div id="R-header-topbar" class="default-animation"></div>
      <div id="R-header-wrapper" class="default-animation">
        <div id="R-header" class="default-animation">
          <a id="R-logo" class="R-default" href="/index.html">
            <div class="logo-title"></div>
          </a>
        </div>
        <search><form action="/search/index.html" method="get">
          <div class="searchbox default-animation">
            <button class="search-detail" type="submit" title="Search (CTRL+ALT+f)"><i class="fas fa-search"></i></button>
            <label class="a11y-only" for="R-search-by">Search</label>
            <input data-search-input id="R-search-by" name="search-by" class="search-by" type="search" placeholder="Search...">
            <button class="search-clear" type="button" data-search-clear="" title="Clear search"><i class="fas fa-times" title="Clear search"></i></button>
          </div>
        </form></search>
      </div>
      <div id="R-homelinks" class="default-animation homelinks">
        <div class="R-menu-divider default-animation">
          <hr class="padding">
        </div>
        <div class="R-sidebarmenu R-shortcutmenu-homelinks">
          <ul class="space collapsible-menu">
            <li class="" data-nav-id="/index.html"><a class="padding" href="/index.html">Home</a></li>
          </ul>
        </div>
        <div class="R-menu-divider default-animation">
          <hr class="padding">
        </div>
        <div class="R-sidebarmenu R-shortcutmenu-headercontrols">
          <ul class="">
          </ul>
        </div>
        <div class="R-menu-divider default-animation">
          <hr class="padding">
        </div>
      </div>
      <div id="R-content-wrapper" class="highlightable">
        <div class="R-sidebarmenu R-shortcutmenu-main">
          <ul class="enlarge morespace collapsible-menu">
            <li class="" data-nav-id="/1-worklog/index.html"><a class="padding" href="/1-worklog/index.html"> <b> 1. </b> Worklog</a><ul id="R-subsections-16447363d800ec5deaa3b0a76c63c922" class="collapsible-menu"></ul></li>
            <li class="parent " data-nav-id="/3-blogstranslated/index.html"><a class="padding" href="/3-blogstranslated/index.html"> <b> 3. </b> BlogsTranslated</a><ul id="R-subsections-dff4ca4b61121a10b58696aacb50b106" class="collapsible-menu">
            <li class="" data-nav-id="/3-blogstranslated/3.1-blog1/index.html"><a class="padding" href="/3-blogstranslated/3.1-blog1/index.html"> <b> 3.1. </b> Blog 1</a></li>
            <li class="" data-nav-id="/3-blogstranslated/3.2-blog2/index.html"><a class="padding" href="/3-blogstranslated/3.2-blog2/index.html"> <b> 3.2. </b> Blog 2</a></li>
            <li class="" data-nav-id="/3-blogstranslated/3.3-blog3/index.html"><a class="padding" href="/3-blogstranslated/3.3-blog3/index.html"> <b> 3.3. </b>Blog 3</a></li>
            <li class="" data-nav-id="/3-blogstranslated/3.4-blog4/index.html"><a class="padding" href="/3-blogstranslated/3.4-blog4/index.html"><b> 3.4. </b> Blog 4</a></li>
            <li class="" data-nav-id="/3-blogstranslated/3.5-blog5/index.html"><a class="padding" href="/3-blogstranslated/3.5-blog5/index.html"><b> 3.5. </b> Blog 5</a></li>
            <li class="" data-nav-id="/3-blogstranslated/3.6-blog6/index.html"><a class="padding" href="/3-blogstranslated/3.6-blog6/index.html"><b> 3.6. </b> Blog 6</a></li>
            <li class="" data-nav-id="/3-blogstranslated/3.7-blog7/index.html"><a class="padding" href="/3-blogstranslated/3.7-blog7/index.html"><b> 3.7. </b> Blog 7</a></li>
            <li class="active " data-nav-id="/3-blogstranslated/3.8-blog8/index.html"><a class="padding" href="/3-blogstranslated/3.8-blog8/index.html"><b> 3.8. </b> Blog 8</a></li></ul></li>
            <li class="" data-nav-id="/4-eventparticipated/index.html"><a class="padding" href="/4-eventparticipated/index.html"> <b> 4. </b> EventParticipated</a><ul id="R-subsections-a3223440c4beb7b035882258d4302894" class="collapsible-menu"></ul></li>
          </ul>
        </div>
        <div class="R-sidebarmenu R-shortcutmenu-shortcuts">
          <ul class="space collapsible-menu">
          </ul>
        </div>
        <div id="R-footer-margin"></div>
        <div class="R-menu-divider default-animation">
          <hr class="padding">
        </div>
        <div class="R-sidebarmenu R-shortcutmenu-footercontrols">
          <ul class="">
			<li class="R-languageswitcher">
              <div class="padding menu-control">
                <i class="fa-fw fas fa-language"></i>
                <span>&nbsp;</span>
                <div class="control-style">
                  <label class="a11y-only" for="R-select-language">Language</label>
                  <select id="R-select-language">
                    <option id="R-select-language-vi" value="vi" data-url="/vi/3-blogstranslated/3.8-blog8/index.html" lang="vi">Ti·∫øng Vi·ªát</option>
                    <option id="R-select-language-en" value="en" data-url="/3-blogstranslated/3.8-blog8/index.html" lang="en" selected data-selected>English</option>
                  </select>
                </div>
                <div class="clear"></div>
              </div>
            </li>
          </ul>
        </div>
<div id="R-footer"><p>Built with <a href="https://github.com/McShelby/hugo-theme-relearn" title="love"><i class="fas fa-heart"></i></a> by <a href="https://gohugo.io/">Hugo</a></p></div>
      </div>
    </aside>
    <script src="/js/clipboard/clipboard.min.js?1762742815" defer></script>
    <script src="/js/perfect-scrollbar/perfect-scrollbar.min.js?1762742815" defer></script>
    <script src="/js/theme.js?1762742815" defer></script>
  </body>
</html>
